mira2_plugin_sparcomulti.i

```diff
@@ -76,32 +77,34 @@ func parse_options(plugin, opt)
   h_set, opt, an_option=my_needed_value;
 
   /* Read SPARCO settings */
-  sparco = opt.sparco_model;
+  sparco = strupper(opt.sparco_model);
   params = opt.sparco_params;
   w0 = opt.sparco_w0;
   image = opt.sparco_image;
-  type = opt.sparco_type;
+  type =   opt.sparco_type;
   file = opt.sparco_file;
   temp = opt.sparco_temp;
   index = opt.sparco_index;
   shift = opt.sparco_xy;
   flux = opt.sparco_flux;
-  spectrum = opt.sparco_spectrum;
-
+  spectrum = strupper(opt.sparco_spectrum);
+  
   if ( !is_void(w0) ) {
-    w0 *= 1e-6;
+    if(w0>0.1){ // assume it is micron
```

Comment:
There is ambiguities about units, this is an ugly patch to not break your previous workflows however I think wavelength should be in SI units as usually in fits files